packages/embark/src/lib/core/events.js

```diff
@@ -4,6 +4,10 @@ const cloneDeep = require('lodash.clonedeep');
 
 const fs = require('fs-extra');
 
+function debugEventsEnabled() {
+  return !!globalThis?.process?.env?.DEBUGEVENTS;
+}
+
```

Comment:
This is actually not valid in the installed typescript version

Reply:
Actually, it's eslint that's complaining on CI

Reply:
I've updated the PR to not make use of the elvis operator here. We can reintroduce this when this is being moved to TS, which in fact is happening in #2016